src/modes/linear_world.cpp

```diff
@@ -262,6 +268,45 @@ void LinearWorld::updateGraphics(float dt)
 
 }   // updateGraphics
 
+// ----------------------------------------------------------------------------
+/** This calculate the time difference between the second kart in the
+ *  race and the first kart in the race (who must be a ghost)
+ */
+void LinearWorld::updateLiveDifference()
+{
+    // First check that the call requirements are verified
+    assert (race_manager->hasGhostKarts() && race_manager->getNumberOfKarts() >= 2);
+
+    AbstractKart* ghost_kart = getKart(0);
```

Comment:
Not a huge fan of hardcoding kart 0 / kart 1. Constants explaining which is which would be better